docs/learning/backend/Fonts.md

```diff
@@ -1,52 +1,86 @@
 
-### TrueType Fonts
+The unit of embedding is a *font file*. Extracting individual font faces from a font file to embed separately is forbidden by most non-free font licenses. The embedding rights for a font file are determined by the most restrictive embedding rights applied to any face within it.
 
-Fonts defined in the `OpenType` format are delivered in files with the extensions: ".ttf", ".ttc", ".otf", or ".otc". They may be imported into Font Workspaces by supplying the name of the  file.
+## Font families
 
-> Note: iTwin.js uses the term `TrueType` where a more accurate term would be `OpenType`. Ignore the distinction.
+A font family is uniquely identified across iModels by its name and type, as encapsulated by [FontFamilyDescriptor]($common). It is possible to have two font families with the same name but of different [FontType]($common)s. Within an iModel, a font family is identified by its unique integer [FontId]($common).
 
-#### TrueType Font Faces
+To use a font family, you must first embed one or more [font files](#font-files) containing the face(s) belonging to that family. Then, you must associate a [FontId]($common) with the [FontFamilyDescriptor]($common). The mapping of family descriptors to font Ids is stored independently from the embedded font files. So, for a given font family, any of the following is possible:
+1. The iModel contains a font Id for the family, and one or more font files containing faces belonging to that family; or
+2. The iModel contains a font Id for the family, but not corresponding font files; or
+3. The iModel contains font files defining faces belonging to that family, but no corresponding font Id; or
+4. No information about the family exists in the iModel.
 
-TrueType fonts may supply separate *faces* to display `Regular`, `Italics`, `Bold`, and `BoldItalics` flavors of the same font name. OpenType files with the extension ".ttf" or ".otf" hold a single font face. OpenType files with the extension ".ttc" or ".otc" hold a *collection* of faces. Sometimes .ttc/.otc collection files hold multiple faces for the same font name and sometimes they hold font faces from multiple fonts.
+Case #1 above is the optimal one, and [IModelDbFonts.embedFontFile]($backend) by default ensures that a font Id is allocated for each font family in the [FontFile]($backend) being embedded. Case #4 can be addressed by embedding the required font file(s). You may encounter cases #2 and #3 when dealing with font information created by older connectors. You can address #2 by embedding the required font file(s), and #3 by using [IModelDbFonts.acquireId]($backend).
 
-## Font Workspaces
+## IModelDbFonts
 
-Fonts may be stored in a [WorkspaceDb]($backend) using the Workspace Editor.
+[IModelDb.fonts]($backend) provides APIs for reading and writing font-related information in an iModel.
```

Comment:
As someone who doesn't know anything at all about the font APIs, I personally was a bit confused by the existence of both the IModelDb.fonts property and the IModelDbFonts type, until I realized that IModelDb.fonts is an instance of IModelDbFonts. It's not necessary, but I think something like the following would avoid such confusion:

> IModelDb.fonts is an instance of IModelDbFonts that provides...